person.rb

```diff
@@ -7,17 +13,52 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    super
```

Comment:
**Perfect implementation on your code!** 

- [ ] However, I'm getting errors if I test your app by running `ruby person.rb`. To fix that I suggest you remove all unused super from your code to make your app approvable!

Reply:
@Rustamxon7  when i remove the super then i got linters error